src/delta/delta.c

```diff
@@ -207,102 +208,110 @@ static int found_override(const char *top, const char *bottom) {
         return k;
 }
 
-static int enumerate_dir_d(Hashmap *top, Hashmap *bottom, Hashmap *drops, const char *toppath, const char *drop) {
-        _cleanup_free_ char *unit = NULL;
-        _cleanup_free_ char *path = NULL;
+static int enumerate_dir_d(Hashmap *bottom, const char *toppath, const char *drop) {
```

Comment:
Yeah, I'll try to rename them to something more descriptive. The "drop" variable is not needed btw, it's just a basename of toppath. In fact, drop isn't even used...